app/translations.coffee

```diff
@@ -14,12 +14,13 @@ translations =
     'searching-position': 'Paikannetaan...'
     'own-position': 'Nykyinen sijaintisi'
     'no-position': 'Ei sijaintia'
-    'origin': 'Mistä? - osoite tai pysäkki'
+    'origin-placeholder': 'Mistä? - osoite tai pysäkki'
+    'origin': 'lähtöpaikka'
     'to-stop': 'Pysäkille'
     'line': 'Linja'
     'leaves': 'Lähtee'
     'next': 'Seuraava'
-    'destination': 'Määränpää'
+    'destination': 'määränpää'
```

Comment:
The first letter can still be capitalized, so that it looks ok in other places, same goes with destination, etc.